src/visualization/plot_recipes.jl

```diff
@@ -107,13 +107,16 @@ PlotData2D(u_ode::AbstractVector, semi; kwargs...) = PlotData2D(wrap_array(u_ode
 
 """
     PlotData2D(sol::TrixiODESolution; kwargs...)
+    PlotData2D(sol::TimeIntegratorSolution; kwargs...)
 
-Create a `PlotData2D` object from an `ODESolution` created by Trixi.
+Create a `PlotData2D` object from an `DiffEqBase.ODESolution` or `TimeIntegratorSolution` created by
+Trixi.
 
 !!! warning "Experimental implementation"
     This is an experimental feature and may change in future releases.
 """
 PlotData2D(sol::TrixiODESolution; kwargs...) = PlotData2D(sol.u[end], sol.prob.p; kwargs...)
+PlotData2D(sol::TimeIntegratorSolution; kwargs...) = PlotData2D(sol.u[end], sol.prob.p; kwargs...)
```

Comment:
Using a `union` above, this will hopefully not be necessary anymore.

Reply:
Removed in 17d22ba7.